test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -369,6 +380,7 @@ context('action sidebar', function() {
 
     cy
       .get('.picklist')
+      .should('not.contain', 'Not Included')
```

Comment:
FYI: I tested this test with `getRoles` in the `patient/action_views` and the test fails. :tada: